efficiency_nodes.py

```diff
@@ -89,12 +89,20 @@ def efficientloader(self, ckpt_name, vae_name, clip_skip, lora_name, lora_model_
         vae_cache, ckpt_cache, lora_cache = get_cache_numbers("Efficient Loader")
 
         if lora_name != "None" or lora_stack is not None:

```

Comment:
This line should actually be:
```
if lora_name != "None" or lora_stack:
```
So we take into account both when lora_stack is not connected (value = None) and when its an empty list [ ]. The current version you posted will give an error if you connect the LoRA Stacker and set all the LoRA names on both Efficient Loader and LoRA Stacker to "None".